DEVDOC.md

```diff
@@ -116,6 +116,42 @@ lookups.
 
 Any help to fix this is very welcome!
 
+#### Integration with VSCode and Delve
+
+If you would like to use the delve debugger for Go with VSCode to debug `gapis`, You can follow these steps
+
+1. Complete the fusing steps.
+
+2. Create a launch.json under the project directory `Ctrl + Shift + P` and `Debug: Open launch.json`
+
+3. Paste this as one of the configuration. This will ensure that you have a setup for attaching to Delve when you launch the project.
```

Comment:
Typo: as one of the configuration**S**. "Launching the project" doesn't make much sense here.